Sources/RemoteNotification.swift

```diff
@@ -33,28 +33,39 @@ public struct RemoteNotification: CustomStringConvertible, Equatable {
     /// The identifier for the UIUserNotificationCategory associated with the push notification. Derived from userInfo["category"]
     public let categoryIdentifier: String?
     /// Custom fields on the remote notification.
-    public let userInfo: [String : AnyObject]
+    public let userInfo: [String : Any]
     /// The dictionary representation of the remote notification.
-    public let remoteNotificationDictionary: [String : AnyObject]
+    public let remoteNotificationDictionary: [String : Any]
```

Comment:
We don't actually know what the expected types are. All we know is that they must be JSON-compatible. Anything you can shove into a push notification can end up here.